### PR TITLE
[android] enable 64-bit in standalone builds for SDK33

### PR DESCRIPTION
**Pull request by wkozyra95**
#4374 

The same as https://github.com/expo/expo/pull/4564, probably vest to merge this closer to sdk34 release

Marking as a draft for now

### CODE REVIEW

File: android/versioned-abis/expoview-abi33_0_0/build.gradle [hunk 1]
```diff
@@ -41,7 +41,7 @@ android {
     versionCode 1
     versionName "1.0"
     ndk {
-      abiFilters 'armeabi-v7a', 'x86'
+      abiFilters 'armeabi-v7a', 'x86', 'arm64-v8a', 'x86_64'
```
Comment (sjchmiela) [inline android/versioned-abis/expoview-abi33_0_0/build.gradle]: Does it make sense to make this change?

File: android/expoview/build.gradle [hunk 2]
```diff
@@ -74,7 +74,7 @@ android {
     versionCode 1
     versionName "1.0"
     ndk {
-      abiFilters 'armeabi-v7a', 'x86'
+      abiFilters 'armeabi-v7a', 'x86', 'arm64-v8a', 'x86_64'
```
Comment (sjchmiela) [inline android/expoview/build.gradle]: Shouldn't we also change something in `ReactAndroid`?

**Review by sjchmiela — COMMENTED** [review]
💬

File: android/expoview/build.gradle (hunk 2 above)
Comment (wkozyra95) [reply android/expoview/build.gradle]: react native is supporting 64 bits so  we don't need to change anything there

File: android/versioned-abis/expoview-abi33_0_0/build.gradle (hunk 1 above)
Comment (wkozyra95) [reply android/versioned-abis/expoview-abi33_0_0/build.gradle]: I assumed that when we run update-exponent-view we are using this build gradle. Am I wrong?

File: expokit-npm-package/maven/host/exp/exponent/expo-ads-admob/5.0.1/ivy-5.0.1.xml [hunk 3]
```diff
@@ -108,7 +108,7 @@
   </publications>
   <dependencies>
     <dependency org="com.android.tools.build" name="aapt2" rev="3.3.2-5309881" conf="_internal_aapt2_binary-&gt;default">
-      <artifact name="aapt2" type="jar" ext="jar" m:classifier="osx" conf="_internal_aapt2_binary"/>
+      <artifact name="aapt2" type="jar" ext="jar" m:classifier="linux" conf="_internal_aapt2_binary"/>
```
Comment (sjchmiela) [inline expokit-npm-package/maven/host/exp/exponent/expo-ads-admob/5.0.1/ivy-5.0.1.xml]: This change doesn't affect anything, right?

**Review by sjchmiela — APPROVED** [review]
LGTM!

**Pull request merged**